applications/GeoMechanicsApplication/tests/cpp_tests/custom_elements/test_geo_steady_state_piping_element.cpp

```diff
@@ -357,7 +357,8 @@ KRATOS_TEST_CASE_IN_SUITE(GeoSteadyStatePwPipingElementReturnsEquilibriumHeightF
                                                                  p_element->GetGeometry(), 0.);
 
     // Assert
-    KRATOS_EXPECT_EQ(pipe_height, 1e10);
+    const double infinite_pipe_height = 1.0e10;
+    KRATOS_EXPECT_EQ(pipe_height, infinite_pipe_height);
```

Comment:
Only to be changed when you also need to make other changes (otherwise we'll take care of it in a next cleaning iteration):
```suggestion
    KRATOS_EXPECT_DOUBLE_EQ(pipe_height, infinite_pipe_height);
```